hadoop-ozone/recon/src/test/java/org/apache/hadoop/ozone/recon/heatmap/TestHeatMapInfo.java

```diff
@@ -965,150 +972,149 @@ public void testHeatMapInfoResponseWithEntityTypeBucket() throws IOException {
-      //                        }
-      //                    ],
-      //                    "size": 107,
-      //                    "minAccessCount": 263,
-      //                    "maxAccessCount": 263
-      //                },
-      //                {
-      //                    "label": "obsbuck12",
-      //                    "path": "/testnewvol2/obsbuck12",
-      //                    "children": [
-      //                        {
-      //                            "label": "",
-      //                            "path": "/testnewvol2/obsbuck12/",
-      //                            "size": 100,
-      //                            "accessCount": 200,
-      //                            "color": 1.0
-      //                        }
-      //                    ],
-      //                    "size": 100,
-      //                    "minAccessCount": 200,
-      //                    "maxAccessCount": 200
-      //                },
-      //                {
-      //                    "label": "obsbuck13",
-      //                    "path": "/testnewvol2/obsbuck13",
-      //                    "children": [
-      //                        {
-      //                            "label": "",
-      //                            "path": "/testnewvol2/obsbuck13/",
-      //                            "size": 100,
-      //                            "accessCount": 200,
-      //                            "color": 1.0
-      //                        }
-      //                    ],
-      //                    "size": 100,
-      //                    "minAccessCount": 200,
-      //                    "maxAccessCount": 200
-      //                }
-      //            ],
-      //            "size": 607
-      //        }
-      //    ],
-      //    "size": 607,
-      //    "minAccessCount": 200,
-      //    "maxAccessCount": 701
-      //}
-      EntityReadAccessHeatMapResponse entityReadAccessHeatMapResponse =
-          heatMapUtil.generateHeatMap(entityMetaDataList);
-      assertThat(entityReadAccessHeatMapResponse.getChildren().size()).isGreaterThan(0);
-      assertEquals(2,
-          entityReadAccessHeatMapResponse.getChildren().size());
-      assertEquals(0.0,
-          entityReadAccessHeatMapResponse.getChildren().get(0).getColor());
-      String path =
-          entityReadAccessHeatMapResponse.getChildren().get(1).getChildren()
-              .get(0).getPath();
-      assertEquals("/testnewvol2/fsobuck11", path);
-    } else {
-      assertNull(entities);
+    JsonNode rootNode = JsonUtils.readTree(auditRespStrWithPathAndBucketEntityType);
+    // Navigate to the nested JSON objects
+    JsonNode facetsNode = rootNode.path("facets");
+    JsonNode resourcesNode = facetsNode.path("resources");
+    // Deserialize the resources node directly if it's not missing
+    HeatMapProviderDataResource auditLogFacetsResources = null;
+//    assumeThat(resourcesNode.isMissingNode()).isFalse();
```

Comment:
Remove this commented code. Pls check other places also.

Reply:
Done!